src/Core/src/Handlers/WebView/WebViewHandler.Windows.cs

```diff
@@ -25,7 +26,7 @@ internal WebNavigationEvent CurrentNavigationEvent
 
 		protected override void ConnectHandler(WebView2 platformView)
 		{
-			platformView.CoreWebView2Initialized += OnCoreWebView2Initialized;
+			_proxy.Connect(this, platformView);
```

Comment:
Hum shouldn't we just connect the proxy  when WebView2 is initialized? I think we need this I remember other bug related with this where there was an exception related with this. @Eilon  do you remember ?

Reply:
The connect method should be the same as what it was doing before:

```csharp
			public void Connect(WebViewHandler handler, WebView2 platformView)
			{
				_handler = new(handler);
				platformView.CoreWebView2Initialized += OnCoreWebView2Initialized;
			}
```
I don't think I changed any events -- just moved methods to a new class.

Reply:
@rmarinho hmm sorry I don't specifically remember a bug around this, but maybe there was.